### ISSUE TITLE
Example doesn`t work in IE 11

### ISSUE THREAD

**Issue by alexua1984** [futurepress/epub.js#144]
Hi, 
Your example doesn't work 
http://futurepress.github.io/epub.js/reader/

**Comment by readinglab2** [comment]
On IE 11, I managed to get reader working with wgxpath. you need call wgxpath.intall(this.render.window) when renderer's content is loaded.

**Comment by DavidPetit** [comment]
I have managed to make wgxpath work but there are still some errors like pelayo "Object does not have evaluate method or function".
We can't navigate on more than the 2 first page and when clicking on the menu button on the top left, the chapters list does not appear
Does anyone know how to fully make it work on IE11 and solve the remaining errors?
If yes, can we see a demo on the version that works on IE11 so that we can inspire to integrate epubjs the same way.

**Comment by readinglab2** [comment]
I didn't get TOC work either because xmldocument is not supported by wgxpath and now have no time to work on it. 

You can check with www.readbook.today to see a live demo. basically you need call wgxpath.install(this.render.window) for every chapter.

**Comment by DavidPetit** [comment]
Ok! Thanks for the information. So at least we can read the pages which is already good enough.
In which function or callback do you put this call for every chapter?

**Comment by benwalsh** [comment]
After installing wgxpath following the example, I still get an `xpath` error at line (5611)[https://github.com/futurepress/epub.js/blob/master/build/epub.js#L5611] where it calls `tocXml.evaluate`.

**Comment by readinglab2** [comment]
sorry, I don't know how to add evaluate to xml document either.

**Comment by pateljatin** [comment]
@readinglab2 - can you please provide a sample as to where you're adding wgxpath.install(this.render.window)?
Is it here:
```
EPUBJS.Hooks.register("beforeChapterDisplay").wgxpath = function (callback, renderer) {
				wgxpath.install(renderer.render.window);
				if (callback) callback();
			};
			wgxpath.install(this.render.window);
```

Thanks!

**Comment by nmuldashev** [comment]
In with file i should add this? I've added this in my player.html file in 
document.onreadystatechange = function() {} but have an error:

 * Unable to get property 'window' of undefined or null reference

I changed it to 
* wgxpath.install(EPUBJS.Render.window);
and got an error 
* Object doesn't support property or method 'querySelector' undefined
* Object doesn't support property or method 'querySelector' undefined
* Could not load book at: META-INF/container.xml
* Object doesn't support property or method 'querySelector' undefined

**Comment by rmunsky** [comment]
see also #50

**Comment by MarkisLt** [comment]
Not good but also a solution. 
If you're using jQuery then in epub.js replace line 5611:
    snapshot = tocXml.evaluate("*[local-name()='navPoint']", parent, null, XPathResult.ORDERED_NODE_SNAPSHOT_TYPE, null),

with:
snapshot = $(tocXml).find('navPoint'),

**Comment by claytonk** [comment]
See https://github.com/futurepress/epub.js/issues/235#issuecomment-143008704 provides TOC fix for IE

**Comment by fchasen** [comment]
Fixed in 0.2.11: https://github.com/futurepress/epub.js/releases/tag/v0.2.11
